docs/development/design-decisions.md

```diff
@@ -5,64 +5,81 @@ This file documents the design choices as well as configuration options.
 ## Stateless
 
 No state storage is needed on `renovate` or the source code repository apart from what you see publicly (branches, Pull Requests).
-It, therefore, doesn't matter if you stop/restart the script and would even still work if you had it running from two different locations, as long as their configuration was the same.
+It, therefore, doesn't matter if you stop/restart the script.
+The script would even still work if you had it running from two different locations, as long as their configuration was the same.
 
 ## Synchronous Operation
 
-The script current processes repositories and branches within them all synchronously.
+The script processes repositories and branches within them synchronously.
+This has the following benefits:
 
 - Greatly reduces the chance of hitting simultaneous API rate limits
 - Simplifies logging
 
 ## Cascading Configuration
 
-Configuration options applied per-package (e.g. with package rules) override those applied per package-type, which override those per-repository, which override those which are global (all repositories).
+We use a cascading configuration.
```

Comment:
I hope this is better than what we had. Please check my list if this is the right order.